src/extensions/scratch3_boost/index.js

```diff
@@ -439,10 +457,11 @@ class BoostMotor {
     }
 
     /**
-     * Turn this motor on indefinitely.
+     * Turn this motor on indefinitely
+     * @private
      */
-    turnOn () {
-        if (this._power === 0) return;
+    _turnOn () {
+        if (this.power === 0) return;
```

Comment:
`this.power === 0;` is distinct from `BoostMotorState.OFF` ?

Reply:
Yeah, `this.power` describes the motor's power between 0-100, where as `BoostMotorState.OFF` is an enumerated value.